pkg/danm/danm.go

```diff
@@ -183,13 +191,115 @@ func extractConnections(args *cniArgs) error {
   return nil
 }
 
+func getResourcePrefix(args *cniArgs, resourceType string)(string,error){
+  confArgs, err := loadNetConf(args.stdIn)
+  if err != nil {
+    return "", errors.New("cannot load CNI NetConf due to error:" + err.Error())
+  }
+  k8sClient, err := createK8sClient(confArgs.Kubeconfig)
```

Comment:
could you re-use the same client we are creating for reading the Pod info: https://github.com/nokia/danm/blob/0ea0698fbd56252f8d303c58283b2e79c26f0a82/pkg/danm/danm.go#L142

in fact, it looks like you copy-pasted this exact function, rather than simply re-using the same client